doc/api/Renter.md

```diff
@@ -211,6 +212,32 @@ lists the status of all files.
 }
 ```
 
+#### /renter/prices [GET]
+
+lists the estimated prices of performing various network operations.
+
+###### JSON Response
+```javascript
+{
+      // The estimated cost of downloading one terabyte of data from the
+	  // network.
+      "downloadterabyte": "1234", // hastings
+
+	  // The estimated cost of forming a set of contracts on the network. This
+	  // cost also applies to the estimated cost of renewing the renter's set of
+	  // contracts.
+      "formcontracts": "1234", // hastings
+
+      // The estimated cost of storing one terabyte of data on the network for
+	  // a month, including accounting for redundancy.
+      "storageterabytemonth": "1234", // hastings
+
+	  // The estimated cost of uploading one terabyte of data to the network,
+	  // including accounting for redundancy.
+      "uploadterabyte": "1234", // hastings
```

Comment:
yes, it's currently hard-coded. We can't look at the renter settings because the renter doesn't have a default setting for file redundancy.

This adds some friction and misunderstanding, but I don't want to mislead people by showing a non-redundant price and then oops-it's-3x-the-price-we-said.